communication/data_logging.cpp

```diff
@@ -42,15 +42,15 @@
 
 #include "communication/data_logging.hpp"
 
+#include <cstdlib>
+#include <math.h>
```

Comment:
Could you clarify if you forget to put math.h back within extern C or if for some reason it can remain up there please ?